templates/api-template-go/README.MD

```diff
@@ -0,0 +1,3 @@
+## jazz api-template-go
+
+An Jazz Template of AWS Lambda with Go.
```

Comment:
It may help if we add more information on how to the tests can be run, including the commands needed.